suite-native/module-passphrase/package.json

```diff
@@ -20,6 +21,7 @@
         "@suite-native/navigation": "workspace:*",
         "@trezor/styles": "workspace:*",
         "react": "18.2.0",
-        "react-native": "0.73.2"
+        "react-native": "0.73.2",
+        "react-native-reanimated": "3.7.0-nightly-20240101-9d365ae7b"
```

Comment:
why is nightly version needed?

Reply:
Not needed for this PR, but it's used everywhere so we want to keep same version as everywhere.

Reply:
Oh, I wasn't aware of this. 👍